_episodes/l1-04-managing_contributions.md

```diff
@@ -0,0 +1,166 @@
+- Some repositories might require the approval of one or more reviewers before the changes can be merged into the target branch. This can be set up by the repository manager(s) as a [branch protection rule](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule).
+- Only administrators of the target repository can merge PR.
+
+### Reviewing a PR
+- When reviewing a PR, you will be shown, for each file changed, a comparison between the old and the new version, much like the `git diff` command (indeed, it is `git diff` between the original and target branches, just nicely formatted).
+- You can add comments and suggest changes to specific lines in the code.
+- Comments and suggestions must be constructive and help the code to become better. Comments of the type “this can be done better” are discouraged. The CONTRIBUTING or the CODE_OF_CONDUCT files often contain information on how to make a good review.
+
+## Issues
+
+- Issues are another feature of GitHub, used to report bugs, request
+ features or enhancements, or to discuss implementation details of some parts
+  of the code. 
+- Issues are a kind of TODO list, with pending and completed tasks as well as
+ serving to prioritise the development activity. 
+- Labels can be added to the issues by the repository administrator to inform
+ at a first glance what the issue is about. Typical labels are "bug",
+  "enhancement", "low priority" or "good first issue", for example.
+- Issues also can have one or more people assigned to them who will take care
+ of sorting them out and closing them when done or if no longer relevant.
+ - By default any GitHub user can create an issue in a public repository
+
+> ## Mentioning other issues and PR
+>
+> All issues and PR receive a tag number starting at 1 and preceded by #, 
+> like #40 or #110. If you want to refer to an issue or PR in any comment
+> anywhere in GitHub, just use its tag number and these will be automatically
+> linked from the comment. 
+ {: .callout}
+
+> ## Claim issues
+>
+> There are some [restrictions][] on who can be assigned to an issue. If you do
+> not have write access to the repository (which is often the case) and you are
+> not part of the same organisation of the repository, the only way of being
+> assigned to an Issue is by making a comment on the Issue. This also serves to
+> warn others that you are volunteering to work on that. A "Hey, I can tackle
+> this." is often enough.
+{: .callout}
+
+[restrictions]: https://help.github.com/en/github/managing-your-work-on-github/assigning-issues-and-pull-requests-to-other-github-users
+
+> ## Closing issues
+>
+> If a PR tackles a particular issue, you can automatically close that issue
+> when the PR is merged by indicating `Close #ISSUE_NUMBER` in any commit
+> message of the PR or in a comment within the PR. 
+ {: .callout}
```

Comment:
For this lesson, this is the only part that I would leave in relation to issues. All of the above was already discussed in the introductory course, so I will just add a link there.

Reply:
Good point. I have now amended this section.